crescita-personale/tirocinio.md

```diff
@@ -0,0 +1,148 @@
+# Tirocinio
+
+In Nebulab, crediamo fermamente nella capacità delle persone di esprimere il proprio potenziale
+quando si trovano nelle condizioni adatte. Per questo abbiamo inserito nel nostro team una posizione
+di tirocinio permanente.
+
+L'obiettivo del tirocinio è quello di inserire sviluppatori alle prime armi nel contesto in cui ci 
+lavoriamo tutti i giorni, col fine di insegnargli le best practice del settore dal punto di vista
+tecnico e delle _soft skill_. Con noi imparerai tutto quello che ti serve per lavorare su progetti
+reali con clienti reali e tutte le responsabilità che ne conseguono.
+
+Il nostro tirocinio dura quattro mesi. Al termine del tirocinio, il tirocinante viene solitamente 
+assunto in Nebulab. Se questo non dovesse essere possibile, lo aiuteremo a trovare una posizione più 
+adatta.
+
+## Selezione
+
+Il processo di selezione è piuttosto semplice e lineare.
+
+### Requisiti base
+
+Innanzitutto, assicurati di possedere i requisiti base per fare domanda, che sono:
+
+- Ottima padronanza dell'italiano (perché è la lingua che parliamo tra noi) e dell'inglese (perché i 
+nostri clienti sono tutti basati negli US)
+- Buona conoscenza dei concetti di base della programmazione (flow control, strutture iterative più 
+comuni) e della programmazione OOP (classi, oggetti ed ereditarietà)
+- Un atteggiamento proattivo al lavoro e all'apprendimento (= non aspettare che ti diciamo cosa fare
+o cosa leggere, ma chiedicelo/trovalo!)
+
+Ci rendiamo conto che questi requisiti potrebbero escludere dei candidati che non hanno esperienza
+con la programmazione ma vorrebbero imparare. Purtroppo, in questo momento insegnare a qualcuno la
+programmazione sarebbe un investimento di tempo troppo grande per noi. Se non hai esperienza con la
+programmazione ma vorresti imparare, ti consigliamo di consultare queste risorse:
+
+### Lettera di presentazione
+
+Completa il modulo che abbiamo sulla nostra pagina [Careers](https://nebulab.it/careers/) indicando 
+nella lettera di presentazione che sei interessato al tirocinio. Raccontaci un po' di te, della tua 
+esperienza (o mancanza di esperienza) col mondo del lavoro e con gli strumenti che usiamo, del tuo 
+approccio all'apprendimento e del perché sei interessato a Nebulab.
+
+Qualcuno del team leggerà la tua lettera e, se pensa che potresti essere un buon candidato, ti 
+scriverà per trovare l'orario più adatto per un colloquio in videoconferenza. A questo colloquio
+saranno presenti almeno due membri del team di Nebulab.
+
+### Colloquio
```

Comment:
Questa parola, per quanto forse sia la più corretta, mi fa strano; probabilmente perché nella visione comune è sempre legato al colloquio di lavoro classico che mi fa pensare che il candidato si presenti già sudato. 

Propongo di mantenere il senso e semplicemente cambiare il termine in qualcosa tipo `Meeting conoscitivo`, `Chiacchierata informale` o `Call di presentazione`. 

Che dici?